staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -81,6 +81,14 @@ type ListMeta struct {
 	// identical to the value in the first response, unless you have received this token from an error
 	// message.
 	Continue string `json:"continue,omitempty" protobuf:"bytes,3,opt,name=continue"`
+
+	// RemainingItemCount is the number of subsequent items in the list which are not included in this
+	// list response. If the list request contained label or field selectors, then the number of
+	// remaining items is unknown and this field will be unset. If the list is complete (either
+	// because it is unpaginated or because this is the last page), then there are no more remaining
+	// items and this field will also be unset.  Servers older than v1.15 do not set this field.
```

Comment:
This should go through at least one version as an alpha field and be flag gated.

You also need to describe what "do not set this field means".

You used the wrong field name in the godoc, it should be `remainingItemCount`.

You need to describe that this field is only set when chunking is requested (do not calling it pagination, that is not the name we use in doc).

Reply:
This should be alpha in 1.15, and I'd like to see a KEP for this and updates to the public doc that describe how this is used (probably update the chunking KEP).

Reply:
I'm not 100% sold on the name, `remaining` is novel in our use.